Source/Modules/d.cxx

```diff
@@ -331,7 +329,7 @@ class D : public Language {
 
     // Also make the target D version available as preprocessor symbol for
     // use in our library files.
-    String *version_define = NewStringf("SWIG_D_VERSION %u", d_version);
+    String *version_define = NewStringf("SWIG_D_VERSION 2");
     Preprocessor_define(version_define, 0);
     Delete(version_define);
```

Comment:
No need to create a temporary `String` now this isn't dynamic - just:

```
Preprocessor_define("SWIG_D_VERSION 2");
```

Reply:
OK